.github/workflows/test.yml

```diff
@@ -35,10 +35,10 @@ jobs:
           # giving us about 6 months of coverage.
           #
           # Minimum supported rust version (MSRV)
-          - "georust/geo-ci:rust-1.63"
+          - "georust/geo-ci:proj-9.2.1-rust-1.70"
```

Comment:
Why not

```suggestion
          - "georust/geo-ci:proj-9.2.1-rust-1.69"
```

?

Reply:
CI was failing with:

>  package `clap_lex v0.5.1` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.63.0

This is actually a failure building a ci dependency, not a crate dependency (cargo-all-features), but I didn't want to spend much time on it.

Reply:
Ah, my old nemesis!